api/src/pcapi/routes/pro/users.py

```diff
@@ -101,26 +99,9 @@ def patch_validate_email(body: users_serializers.ChangeProEmailBody) -> None:
         users_api.change_pro_user_email(
             current_email=token.data["current_email"], new_email=token.data["new_email"], user_id=token.user_id
         )
-    except users_exceptions.InvalidToken:
-        # TODO (yacine-pc) this bloc is maintained to support existed tokens, it will be removed in PC-25024
-        try:
-            payload = users_serializers.ChangeEmailTokenContent.from_token(body.token)
-            users_api.change_pro_user_email(
-                current_email=payload.current_email, new_email=payload.new_email, user_id=payload.user_id
-            )
-        except pydantic_v1.ValidationError as exc:
-            errors.add_error("global", "Adresse email invalide")
-            raise errors from exc
-        except InvalidTokenError as exc:
-            errors.add_error("global", "Token invalide")
-            raise errors from exc
-        except users_exceptions.UserDoesNotExist as exc:
-            errors.add_error("global", "Token invalide")
-            raise errors from exc
-        except users_exceptions.EmailExistsError:
-            # Returning an error message might help the end client find
-            # existing email addresses.
-            pass
+    except users_exceptions.InvalidToken as exc:
+        errors.add_error("global", "Token invalide")
+        raise errors from exc
     except users_exceptions.UserDoesNotExist as exc:
         errors.add_error("global", "Token invalide")
         raise errors from exc
```

Comment:
Du coup je pense qu’on peut faire

```suggestion
    except (users_exceptions.InvalidToken, users_exception.UserDoesNotExist) as exc:
        errors.add_error("global", "Token invalide")
        raise errors from exc
```